server/validation.js

```diff
@@ -238,8 +238,8 @@ const ParticipantBatchSchema = yup.array().of(
       // Contact info
       firstName: yup.string().required(errorMessageIndex(index, indexName)),
       lastName: yup.string().required(errorMessageIndex(index, indexName)),
-      phoneNumber: yup.mixed().required(errorMessageIndex(index, indexName)),
-      emailAddress: yup.string().required(errorMessageIndex(index, indexName)).matches(/^(.+@.+\..+)?$/, `Invalid email address (row ${index})`),
+      phoneNumber: yup.string().matches(/^([0-9]{10})$/, { excludeEmptyString: true, message: `Phone number must be provided as 10 digits (index ${index})` }).nullable(errorMessageIndex(index)),
```

Comment:
This should be required, not nullable I think?

Reply:
I think we want to follow what's used for single participant POST vie the PEOI form:

`phoneNumber: yup.string().required(errorMessage).matches(/^\d{10}$/, 'Phone number must be provided as 10 digits'),`